R/corPFSOS.R

```diff
@@ -194,7 +194,7 @@ p11 <- function(transition, s, t) {
 #' transition <- exponential_transition(h01 = 1.2, h02 = 1.5, h12 = 1.6)
 #' PFSOSInteg(1, 2, transition)
 PFSOSInteg <- function(u, t, transition) {
-  p11(transition, u, t / u) * survPFS(transition, u) * haz(transition, u, 1)
+  exp(log(p11(transition, u, t / u)) + log(survPFS(transition, u)) + log(haz(transition, u, 1)))
```

Comment:
this is the right direction, but will only help when we avoid initial `exp()` in `p11()`, e.g. consider just directly returning the log in there and rename to `log_p11()`.
The other idea (not for now but for future) would be to allow `survPFS(..., log = TRUE)` and then use that here.

Reply:
Thx @danielinteractive , I fixed it now. Generally speaking, what is the advantage here of using `log_p11()`?